docs/source/quickstart.rst

```diff
@@ -23,13 +90,14 @@ Process 1
     import cupy as cp
 
     n_bytes = 2**30
-    host = ucp.get_address(ifname='enp1s0f0')
+    host = ucp.get_address(ifname='enp1s0f0')  # ethernet device name
```

Comment:
Why is it `eth0` for NumPy and `enp1s0f0` for CuPy? Could they use the same `ifname` or is there a reason for them to be different?